app/assets/javascripts/components/common/feedback.jsx

```diff
@@ -31,16 +46,25 @@ const Feedback = React.createClass({
     }
 
     let modal;
+    let messages = this.props.feedback[this.props.assignment.article_id];
+    const feedbackList = [];
+    if (!messages) {
+      messages = [{ message: 'The Article doesn\'t exist' }];
+    }
+
+    for (let i = 0; i < messages.length; i++) {
+      feedbackList.push(<li>{messages[i].message}</li>);
+    }
     if (!this.state.show) {
       modal = <div className="empty" />;
     } else {
       modal = (
         <div className="article-viewer feedback">
           <h2>Feedback</h2>
```

Comment:
**Styling suggestion:** You could add padding-bottom to this tag, for proper spacing between header and feedback details.